build-tools/biome.json

```diff
@@ -0,0 +1,56 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.5.3/schema.json",
+	"files": {
+		"ignoreUnknown": true,
+		"ignore": ["**/_api-extractor-temp/**", "**/oclif.manifest.json", "**/_package.json"]
+	},
+	"vcs": {
+		"enabled": true,
+		"clientKind": "git",
+		"useIgnoreFile": true
+	},
+	"organizeImports": {
+		"enabled": false
+	},
+	"linter": {
+		"enabled": false,
+		"rules": {
+			"recommended": true
+		}
+	},
+	"formatter": {
+		"enabled": true,
+		"formatWithErrors": true,
+		"indentStyle": "tab",
+		"lineWidth": 95,
+		"lineEnding": "lf"
+	},
+	"javascript": {
+		"formatter": {
+			"arrowParentheses": "always",
+			"jsxQuoteStyle": "double",
+			"semicolons": "always",
+			"trailingComma": "all",
+			"quoteProperties": "asNeeded",
+			"quoteStyle": "double",
+			"bracketSpacing": true,
+			"bracketSameLine": false
```

Comment:
This sounds like the cause of the formatting I disliked in the other comment 😄 was it chosen for some particular reason?

Reply:
[The docs](https://biomejs.dev/reference/configuration/#javascriptformatterbracketsameline) make it sound like this only affects JSX, and unfortunately changing the setting has no effect on the code you noted.

I agree that this is unfortunate. I'd prefer they break on the `extends` if the line max is surpassed, rather than break up the types like that. We can file a bug if we can get a good playground example.